packages/sample-app/sample_app/strands_example.py

```
@@ -0,0 +1,410 @@
+    # Configure LiteLLM model
+    model = LiteLLMModel(
+        model_id="gpt-4o-mini",
+        params={"max_tokens": 1500, "temperature": 0.7}
+    )
+    
+    # Research Agent - Now with real web search capabilities
+    research_agent = Agent(
+        model=model,
+        tools=[research_destination, get_weather_forecast, search_web, get_memory_context],
+        system_prompt="You are a Travel Research Specialist with access to real-time web data. Your expertise includes:\n" +
+                     "- Live destination research using current web information\n" +
```

Flake8 (7.2.0)

[error] continuation line unaligned for hanging indent

(E131)